docs/zkapps/how-zkapps-work.mdx

```diff
@@ -24,153 +24,139 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
+After you write your smart contract, run the `npm run build` command. The build process compiles the TypeScript code into JavaScript and outputs the `smart_contract.js` file. 
 
-- Run your prover function to run your smart contract
-- Generate a verification key to deploy your smart contract
+From this file, you can:
 
-The **prover function** runs in a user’s web browser.
+- Run a prover function to run your smart contract
+- Generate a verification key to deploy your smart contract
 
-The **verification key** lives on chain for a given zkApp account and is used by the Mina network to verify that a zero-knowledge proof has met all constraints defined in the prover. A verification key is required in order to create a zkApp account. 
+While the prover function runs in a user's web browser, the verification key lives on-chain for a given zkApp account and is used by the Mina network to verify that a zero knowledge proof has met all constraints defined in the prover. A verification key is required to create a zkApp account.
 
-The next two sections go into more detail about how the prover and verifier functions are used when a zkApp is deployed and when users interact with a zkApp.
+## Deploy a smart contract
 
-### Deploying a smart contract
+Smart contracts are deployed to the Mina network using the Mina zkApp CLI. The deployment process sends a transaction that contains the verification key to an address on the Mina blockchain.
 
 <figure>
   <img src="/img/6_zkApps_DeploySmartContract.jpg" width="75%" alt="Diagram showing deployment of a Mina zkApp smart contract" />
 </figure>
 
-<br/>
-
-To deploy a smart contract to the Mina network, the zkApp developer uses the Mina zkApp CLI. The deployment process sends a transaction that contains the verification key to an address on the Mina blockchain.
+When a Mina address contains a verification key, it acts as a zkApp account.
 
-When a Mina address contains a verification key, it acts as a **zkApp account**.
-Whereas a regular Mina account can receive any transactions, a zkApp account can only successfully receive transactions that contain a proof that satisfies the verifier function. Any transactions that do not pass the verifier function are rejected by the Mina network.
+A regular Mina account can receive any transactions. 
 
-:::info
-
-When you deploy to a new Mina address, the Mina Protocol charges a 1 MINA fee for account creation. This fee is unrelated to zkAapps and helps to prevent Sybil or denial of service attacks.
-
-:::note
-
-### Deploying a zkApp UI
+Because zkApps are Mina Protocol's smart contracts powered by zero knowledge proofs, a zkApp account can successfully receive only transactions that contain a proof that satisfies the verifier function. Any transactions that do not pass the verifier function are rejected by the Mina network.
```

Comment:
I don't think this is 100% correct - you can specify custom permissions on smart contract accounts and by doing so, you can also send transactions to it that are not authorized by a proof